### ISSUE TITLE
Error parsing tnsnames.ora file in thin mode with Easy Connect String entries

### ISSUE THREAD

**Issue by sosoba** [oracle/node-oracledb#1644]
1. What versions are you using?

6.3.0

2. Is it an error or a hang or a crash?

Error

3. What error(s) or behavior you are seeing?

```
TypeError: Cannot read properties of null (reading 'name')
    at createNode (oracledb\lib\thin\sqlnet\connStrategy.js:167:22)
    at resolveAddress (oracledb\lib\thin\sqlnet\networkSession.js:96:10)
```

4. Include a runnable Node.js script that shows the problem.

`tnsnames.ora`:
```properties
XE = tcp://localhost/XE
```

`test.js`
```js
import oracledb from 'oracledb';
await oracledb.getConnection({connectionString: 'XE', user: 'nobody', password: 'empty'});
```

The connection is established correctly via oracledb thick mode same as sqlplus, SqlDeveloper, ojdbc.

**Comment by sharadraju** [comment]
Thanks @sosoba . We will look at it

**Comment by sreguna** [comment]
Though this works in thick mode, Easy Connect entries in tnsnames.ora is not a documented feature. We will look into adding this support for thin mode.

**Comment by sharadraju** [comment]
This has been fixed in the 6.5 release.
